packages/neon-core/src/sc/OpToken.ts

```diff
@@ -47,9 +53,7 @@ export class OpToken {
     if (opToken.code >= 0 && opToken.code <= 5) {
       //PUSHINT*
       // We dont verify the length of the params. Its screwed if its wrong
-      return opToken.params
-        ? HexString.fromHex(opToken.params, true).toNumber()
-        : 0;
+      return opToken.params ? parseInt(reverseHex(opToken.params), 16) : 0;
```

Comment:
Give me a second , I'm about to look at what the effect of this is

Reply:
This is just a refactor that removes the need to use HexString which is a class vs using direct methods. Should be equivalent. Just using a lower level import.

Reply:
Sorry took a bit longer because I had modification in my C# environment that was giving me weird output.

Following the C# [unit tests](https://github.com/neo-project/neo-vm/blob/master/tests/neo-vm.Tests/Tests/OpCodes/Push/PUSHINT8_to_PUSHINT256.json#L47), the following should output `-2`, instead it gives `65534`

```javascript
const token = Neon.sc.OpToken.fromScript("01feff");
let value = Neon.sc.OpToken.parseInt(token[0]);
console.log(value);
```

Values you use to test
PUSHINT8,  00ff, expected: -1
PUSHINT16, 01feff, expected: -2
PUSHINT32, 03fdffffff, expected: -3
PUSHINT64, 04fcffffffffffffff, -4
PUSHINT128, 05fbffffffffffffffffffffffffffffff, -5
PUSHINT256, 06faffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff, -6

Reply:
fwiw; this `Buffer.from('feff', 'hex').readIntLE(0,2)` produces the correct number

Reply:
looks like i have to work on getting the big integer representation in first for this to be implemented well.

Reply:
can you share some info on what `getting the big integer representation in first` this means? Do you need to find a biginteger library or? If so, we might consider wrapping this [C++ port](https://github.com/CityOfZion/BigIntegerCpp) of the C# Biginteger class. It was specifically made to produce the identical output.

Reply:
I was looking at https://github.com/NeoResearch/csbiginteger-js.

The C++ port is not suitable as that cannot be packaged up for web. Alternatively is just writing some small wrapping functions for reading hexstrings as two complements.

Reply:
I don't want to down talk their work, but I've historically tried to use their biginteger library in python and it produced wrong output in certain cases. What I remember is one option just wrapped a different C++ library, the other had some interface to talk to C# but now you need both .NET and the bindings. For these reasons I did a full native port with 0 dependencies. I'm positive it can be packaged up for web because the javascript's core is C++.

That aside if you think the existing code just needs a couple of small functions, then it's probably not worth the effort to create bindings at this moment.

Reply:
updated to use BigInteger.

Reply:
will check soon

Reply:
It does not seem to like 
```javascript
const token = Neon.sc.OpToken.fromScript("06faffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
```
throwing
```
/Users/erik/Documents/code/neon-js/packages/neon-js/dist/index.js:30659
            const paramsExtracter = annotation.operandSize
                                               ^

TypeError: Cannot read property 'operandSize' of undefined
```

while the following does work as expected
```javascript
const token = Neon.sc.OpToken.fromScript("05fbffffffffffffffffffffffffffffff");
```

Reply:
The rest looks good to me.

Reply:
I believe PUSHINT256 is `05`. The test case is here: https://github.com/CityOfZion/neon-js/pull/585/files#diff-2a331f7ce64869df2deb51b6707975e5R117

Reply:
You're right, my bad